core/storage/base_model/gae_models.py

```diff
@@ -295,34 +300,35 @@ def _reconstitute_from_snapshot_id(self, snapshot_id):
 
     @classmethod
     def _get_snapshot_id(cls, instance_id, version_number):
-        """Gets snapshot id.
+        """Gets unique id for this instance and version.
 
         Args:
-            instance_id: int.
-            version_number: str.
+            instance_id: str.
+            version_number: int.
 
         Returns:
-            str.
+            str. Combination of instance_id, delimiter and version_number.
```

Comment:
Clients don't care about what this id looks like; that is an implementation detail. I would just say "The unique snapshot id corresponding to the given instance and version."